app/mobile/lib/view/course/coursepage.dart

```diff
@@ -265,3 +318,37 @@ class CoursePageViewModel extends ChangeNotifier {
     notifyListeners();
   }
 }
+
+createDiscussion(BuildContext context, CoursePageViewModel viewModel) {
```

Comment:
Can you please separate this to another file alongside the part in the TabBarView? It would be better for the management of the code I think.